package.json

```diff
@@ -33,23 +33,23 @@
   ],
   "dependencies": {
     "agenda": "0.9.0",
-    "azure-arm-compute": "0.20.0",
-    "azure-storage": "2.4.0",
+    "azure-arm-compute": "4.0.0",
+    "azure-storage": "2.8.3",
     "basic-auth": "1.0.4",
     "bluebird": "3.4.0",
     "body-parser": "1.15.1",
     "connect-timeout": "1.9.0",
     "dockerode": "2.2.10",
-    "etcd3": "0.2.9",
+    "etcd3": "0.2.10",
```

Comment:
We don't use it anymore. Please check and remove this.

Reply:
Done!